test/integration/basic_test.go

```diff
@@ -132,6 +137,41 @@ func (m *Minishift) executingMinishiftCommandSucceeds(command string) error {
 	if lastCommandOutput.ExitCode != 0 || len(lastCommandOutput.StdErr) != 0 {
 		return fmt.Errorf("Command did not execute successfully. cmdExit: %d, cmdErr: %s", lastCommandOutput.ExitCode, lastCommandOutput.StdErr)
 	}
+	return nil
+}
+
+func (m *Minishift) commandReturnEquals(commandField, command, expected string) error {
+	m.executingCommand(command)
+	return compareExpectedWithActualEquals(expected+"\n", selectFieldFromLastOutput(commandField))
+}
+
+func (m *Minishift) commandReturnContains(commandField, command, expected string) error {
+	m.executingCommand(command)
+	return compareExpectedWithActualContains(expected, selectFieldFromLastOutput(commandField))
+}
+
+func (m *Minishift) configContains(configPath, expectingResult, expectedKeyPath, expectedValue string) error {
+	data, err := ioutil.ReadFile(testDir + "/" + configPath)
+	if err != nil {
+		return fmt.Errorf("Cannot read config file")
+	}
+	var values map[string]interface{}
+	json.Unmarshal(data, &values)
+	actualValue := ""
+	keyPath := strings.Split(expectedKeyPath, ".")
+	for _, element := range keyPath {
```

Comment:
Yes, this is to be prepared for nested values. For now nested values are not used, but I tested this on OpenShift's config so it should be probably ready for scenarios for `minishift openshift config xxx`. Wanted to make this really reusable.